src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -186,6 +193,41 @@ public function getColumnStyle($columnIndex)
         return $this->getStyle();
     }
 
+    /**
+     * Sets the minimum width of a column.
+     *
+     * @param int $columnIndex Column index.
```

Comment:
I recommend somebody does that for the setColumnStyle method too, so the result is more consistent.